babel/src/test/java/org/apache/calcite/test/BabelParserTest.java

```diff
@@ -2255,4 +2255,51 @@ private void checkParseInfixCast(String sqlType) {
         + "VALUES (ROW(NULL, NULL, NULL))";
     sql(sql).ok(expected);
   }
+
+  @Test public void testHostVariableExecPositionalParams() {
+    final String sql = "exec foo (:bar, :baz, :qux)";
+    final String expected = "EXECUTE `FOO` (:BAR, :BAZ, :QUX)";
+    sql(sql).ok(expected);
+  }
+
+  @Test public void testHostVariableExecNamedParams() {
+    final String sql = "exec foo (bar=:bar, baz=:baz, qux=:qux)";
+    final String expected = "EXECUTE `FOO` (`BAR` = :BAR, `BAZ` = :BAZ,"
+        + " `QUX` = :QUX)";
+    sql(sql).ok(expected);
+  }
+
+  @Test public void testHostVariableSelect() {
+    final String sql = "select :bar from foo where a = :qux";
```

Comment:
nit: Can we add an alias after ":bar" just to cover that case too?

Reply:
Sure I can add that